bucket/privatezilla.json

```diff
@@ -0,0 +1,23 @@
+{
+    "version": "0.43.0",
+    "description": "Performs a privacy & security check of Windows 10",
+    "homepage": "https://www.builtbybel.com/ms-apps/privatezilla",
+    "license": "MIT",
+    "url": "https://github.com/builtbybel/privatezilla/releases/download/0.43.0/privatezilla.zip",
+    "hash": "c161a1b19ac9d4f5c2d6e5ce7d5582b9fd3666debc033685f4f9063cb219c91c",
+    "bin": "Privatezilla.exe",
```

Comment:
Add pre_install check for windows version https://github.com/builtbybel/privatezilla#system-requirements